apertium-get.py

```diff
@@ -0,0 +1,388 @@
+#!/usr/bin/python3
+
+import argparse
+import os
+import re
+import sys
+import urllib.request
+from subprocess import (
+    run,
+    check_call,
+    check_output,
+    STDOUT,
+    DEVNULL,
+    CalledProcessError,
+)
+from enum import Enum
+
+### Globals:
+
+PMODULES = ["trunk", "staging", "nursery", "incubator"]
+LMODULES = ["languages", "incubator"]
+
+git_ssh = "git@github.com:"
+git_https = "https://github.com/"
+
+apertium_git = "apertium/apertium-%s.git"
+giella_git = "giellalt/lang-%s.git"
+giella_core_git = "giellalt/giella-core.git"
+giella_shared_git = "giellalt/giella-shared.git"
+
+
+class Status(Enum):
+    NOT_STARTED = 0
```

Comment:
Ideally, we use https://docs.python.org/3/library/enum.html#using-automatic-values

Reply:
it works for me but results in an import error for Travis

Reply:
ok, `Enum` requires py3.4 while `auto()` requires py3.6